backend/cms_export.json

```diff
@@ -0,0 +1,7 @@
+System check identified some issues:
+
+WARNINGS:
+[33;1m?: (map.W001) Usage restricted tile server.
+	HINT: You are using an open street map tile server for viewing the map, which is subject to usage restrictions (See https://operations.osmfoundation.org/policies/tiles/).
+If you plan to put this system into production, please consider setting up your own tile server or using a commercial service, e.g. mapbox.We readily provide an integration for mapbox, which you can use by setting LEAFLET_TILESERVER='mapbox' and adding your API key by setting the environment variable MAPBOX_TOKEN.[0m
```

Comment:
:D :see_no_evil:
I know this is generated automatically, but should this stay? :sweat_smile:

Reply:
Oh no, I forgot to remove this. That obviously breaks the import

Reply:
Why is this there? Are checks run on dumpdata?
Can you check if they are on STDOUT or STDERR? We might need to adapt the backup script for this.

Reply:
> Are checks run on dumpdata?

Yes

Reply:
Very important, but not directly related to this issue, so I opened a new one: #117